### PR TITLE
added code to allow randomized choice options

### PR DESCRIPTION
**Pull request by pmarcowski**

### CODE REVIEW

File: rlfit.m [hunk 1]
```diff
@@ -13,6 +14,9 @@
 % lb and ub are vectors of upper and lower bounds on parameters
 %
 % niter (optional) is the number of random restarts to use in fitting
+%
+% ispresentx is a vector, one per trial, indicating options present
```
Comment (jmxpearson) [inline rlfit.m]: I think it would clarify things if you noted that it's Noptions x Ntrials (since vectors in Matlab can be row or column).

**Review by jmxpearson — CHANGES_REQUESTED** [review]
Sorry this took so long. Thanks for the PR!

**Comment by pmarcowski** [comment]
Good call, thank you. I changed the description on that one.

File: rlfit.m (hunk 1 above)
Comment (pmarcowski) [reply rlfit.m]: Good call, thank you. I changed the description on that one.

**Pull request merged**